bp_common/src/include/bp_common_aviary_defines.vh

```diff
@@ -245,6 +249,7 @@ typedef struct packed
   , localparam cce_block_width_p          =  `BSG_MAX(dcache_block_width_p,                        \
                                                      `BSG_MAX(icache_block_width_p,                \
                                                        acache_block_width_p))                      \
+  , localparam lce_fill_width_p           =  `BSG_MAX(dcache_fill_width_p, icache_fill_width_p)    \
```

Comment:
We want to support different fill widths for I$ and D$ for LCE

Reply:
So that’s similar to the UCE having a generic fill_width parameter, just in the LCE/LCE submodules